src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,69 @@
+package duke;
+
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Ui class that handles the User interface, that the user sees.
+ */
+public class Ui {
+    public Ui() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        System.out.println(logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+    }
+
+    public void customMessage(String message) {
+        System.out.println(message);
+    }
+    public void customMessage(Task value) {
+        System.out.println(value);
+    }
+
+    public String readCommand() throws IOException {
+        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+        String value = br.readLine();
+        return value;
+    }
+    public void showMarkSucess(Task value) {

```

Comment:
Consider rectifying the spelling/gramatical error here to showMarkSuccessful or maybe just change it to showMarked, makes it less lengthy yet concise. Also, isn't there supposed to be an unmark command too?